lambdas/service/app.py

```diff
@@ -1354,10 +1354,14 @@ def get_project_data(project_id: Optional[str] = None) -> JSON:
                 `fileTypeSummaries` provides the count and total size in bytes
                 of files grouped by their format, e.g. "fastq" or "matrix."
                 `fileCount` and `totalFileSize` compile these figures across all
-                file formats. Likewise, `cellCountSummaries` counts cells and
-                their associated documents grouped by organ type, with
-                `totalCellCount` compiling cell counts across organ types and
-                `organTypes` listing all referenced organs.
+                file formats.
+
+                `cellCountSummaries` provides cell suspension cell counts and
+                the number of associated documents grouped by organ type, with
+                `organTypes` listing all referenced organs. `projects`
```

Comment:
`projects` will have other summaries as well, not just cell counts. The structure is generally `outer_entity.inner_entity.property`.

I don't want to get hung up on this right now, just remove the change and file a FIXME for #3917, please.

Reply:
Created https://github.com/DataBiosphere/azul/issues/4016 to address the missing API documentation of the summary response field `projects`.

Reply:
I meant to ask you to add a FIXME referring to #3917. No new issue is needed.